frontend/src/pages/add-blog.tsx

```diff
@@ -236,35 +200,48 @@ function AddBlog() {
-            <input
-              type="url"
-              id="imagelink"
-              name="imageLink"
-              placeholder="https://&hellip;"
-              autoComplete="off"
-              className="dark:text-textInField w-3/4 rounded-lg bg-slate-200 p-3 placeholder:text-sm placeholder:text-light-tertiary dark:bg-dark-field dark:placeholder:text-dark-tertiary lg:w-10/12"
-              value={formData.imageLink}
-              onChange={handleInputChange}
-            />
-            <button
-              name="openModal"
-              type="button"
-              className="lg:text-md active:scale-click w-1/4 rounded-lg bg-light-primary text-xs text-slate-50 hover:bg-light-primary/80 dark:bg-dark-primary dark:text-dark-card dark:hover:bg-dark-secondary/80 sm:text-sm lg:w-2/12 lg:px-4 lg:py-3"
-              onClick={() => {
-                setmodal(true);
-              }}
-            >
-              Pick image
-            </button>
+          <div>
+            <div className="mb-1 flex justify-between gap-2 sm:gap-4">
+              <input
+                {...register('imageLink')}
+                type="url"
+                id="imagelink"
+                name="imageLink"
+                placeholder="https://&hellip;"
+                autoComplete="off"
+                className="dark:text-textInField w-3/4 rounded-lg bg-slate-200 p-3 placeholder:text-sm placeholder:text-light-tertiary dark:bg-dark-field dark:text-dark-textColor dark:placeholder:text-dark-tertiary lg:w-10/12"
+                value={formData.imageLink}
+              />
+              <button
+                name="openModal"
+                type="button"
+                className="lg:text-md active:scale-click w-1/4 rounded-lg bg-light-primary text-xs text-slate-50 hover:bg-light-primary/80 dark:bg-dark-primary dark:text-dark-card dark:hover:bg-dark-secondary/80 sm:text-sm lg:w-2/12 lg:px-4 lg:py-3"
+                onClick={() => {
+                  setmodal(true);
+                }}
+              >
+                Pick image
+              </button>
+            </div>
+            {errors.imageLink && (
+              <span className="p-2 text-sm text-red-500">{`${errors.imageLink.message}`}</span>
+            )}
           </div>
+
           <div className="mb-4 flex flex-col">
             <label className="px-2 pb-1 font-medium text-light-secondary dark:text-dark-secondary sm:mr-4 sm:w-fit">
               Categories
-              <span className="text-xs tracking-wide text-dark-tertiary">
+              <span
+                {...register('categories')}
+                className="text-xs tracking-wide text-dark-tertiary"
+              >
                 &nbsp;(max 3 categories)&nbsp;
               </span>
               <Asterisk />
+              <br></br>
+              {errors.categories && (
+                <span className="text-sm text-red-500">{`${errors.categories.message}`}</span>
```

Comment:
The error section of the categories should go below the categories selection ig, because in every other thing error is below, only in the cateogry it is above the actual category section

Reply:
ok sir

Reply:
Done sir